CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2.0.1
+
+- Bump the all-dependencies group across 1 directory with 2 updates in [#345](https://github.com/grafana/grafana-amazonprometheus-datasource/pull/345)
+- Migrate to plugin-ui in [#343](https://github.com/grafana/grafana-amazonprometheus-datasource/pull/343)
+- Dependencies: Update @grafana/prometheus and promlib with support for UTF8 metrics and labels in [#339](https://github.com/grafana/grafana-amazonprometheus-datasource/pull/339)
```

Comment:
Wonder if we should increment a minor version, since this is a new feature?

Reply:
Good shout! I've updated it.